mitodl/ocw-www
# add testimonials to home page

#### Pre-Flight checklist


#### What are the relevant tickets?

part of #5 

#### What's this PR do?

- Adds a data model (with a hugo archetype) for the testimonial 
- adds the four testimonials shown in the mockup
- adds a UI to display them on the homepage

#### How should this be manually tested?

visit the home page and check out the little testimonials. they should look like the designs, and you should be able to hover to see a 'lead' quote for each student (taken from what was featured on their entries on https://ocw.mit.edu/about/ocw-stories/ocw-stories-archive/).

I didn't add the links to the full store yet, will do that in a follow-up PR.

#### Screenshots (if appropriate)

![Screenshot from 2021-01-11 11-10-05](https://user-images.githubusercontent.com/6207644/104206632-c745be00-53fd-11eb-8529-368bfb9118e6.png)

this shows the hover state for two of them:

![Screenshot from 2021-01-11 11-10-35](https://user-images.githubusercontent.com/6207644/104206648-cb71db80-53fd-11eb-99de-4c56c88ce00f.png)
